### ISSUE TITLE
Vagrant script for ubuntu fails when run in an isolated environment

### ISSUE THREAD

**Issue by neomatrix369** [AdoptOpenJDK/openjdk-infrastructure#64]
Standing up an environment using `vagrant` (for ubuntu 14.04) and running the [ubuntu.yml](https://github.com/AdoptOpenJDK/openjdk-infrastructure/blob/master/ansible/playbooks/ubuntu.yml) Ansible script halts with the below message:

```
fatal: [localhost]: FAILED! => {"failed": true, "msg": "An unhandled exception occurred while
running the lookup plugin 'file'. Error was a <class 'ansible.errors.AnsibleError'>, 
original message: could not locate file in lookup: /Vendor_Files/keys/id_rsa.pub"})
```

This occurs when run on a local machine (reproducible on Linux and MacOSX environments).

Re-running with `-v` flags will help, `-vv`, `-vvv`, etc... will give will more verbose info about the issue.

See https://github.com/AdoptOpenJDK/openjdk-infrastructure/pull/58#issuecomment-347149275 in https://github.com/AdoptOpenJDK/openjdk-infrastructure/pull/58

Add any additional info to https://github.com/AdoptOpenJDK/openjdk-infrastructure/blob/master/ansible/README.md, once resolved or any findings during the course of the investigation.

#helpwanted #bug

**Comment by bblondin** [comment]
The playbook assumes its running from the master....
thus /Vendor_Files refers to a folder on the AWX server (Ansible Tower)

When running stand alone you need to override this using the extra variables
```
---
Jenkins_User_SSHKey: /some_path/id_rsa.pub
```

You may also wish to disable other options
```
---
Jenkins_Username: MY_USER
Jenkins_User_SSHKey:  /some_path/id_rsa.pub
Nagios_Plugins: Disabled
Slack_Notification: Disabled
Superuser_Account: Disabled
```

**Comment by neomatrix369** [comment]
@bblondin for your feedback, will use the above to apply fix for the failing issue

**Comment by neomatrix369** [comment]
@karianna with the vagrant training over the unconf weekend, you can easily tackle this ;), i'll pair with you if you like :p

**Comment by karianna** [comment]
@neomatrix369 I'll take you up on that next hangout

**Comment by neomatrix369** [comment]
@karianna I think issue is resolved but we can go over this tomorrow or next week over hangout, paste PR link here

**Comment by neomatrix369** [comment]
@sxa555 I have resolved the issue via #66, using `--skip-tags`, close to what you meant?
